lib/config/defaults.json

```diff
@@ -1,5 +1,6 @@
 {
   "server": {
-    "port": 3000
+    "port": 3000,
+    "routesFolder": "lib/routes"
```

Comment:
lib - this name confused me. But if we are ok with it - I'll skip this moment in future

Reply:
What name of the folder should we use? if we will use `app` name we have import like `./app/services/app` when import app service.

Reply:
nevermind... skip this - I'll refactor this code